src/components/widgets-settings/params-picker/params-picker.directive.coffee

```diff
@@ -39,9 +38,11 @@ module.directive('settingParamsPicker', ($templateCache) ->
       deferred: '='
       param: '@',
       options: '=',
+      description: '@',
```

Comment:
should be '@?' to let the developers use the directive without having to specify a custom description